cmd/edgex-secrets-config/README.md

```diff
@@ -0,0 +1,193 @@
+% edgex-secrets-config-proxy(1) User Manuals edgex-secrets-config-proxy(1)
+
+# NAME
+
+edgex-secrets-config-proxy – Configure EdgeX proxy service
+
+# SYNOPSIS
+
+**edgex-secrets-config proxy** SUBCOMMAND [OPTIONS]
+
+# DESCRIPTION
+
+Configures the EdgeX proxy service.
```

Comment:
Remove this line?

Reply:
K, re-read an not under stand `EdgeX proxy service` isn't `proxy setup`... ;-) Don't we typically refer to this as the `EdgeX API Gateway`?

How about `Configures the proxies for the EdgeX API Gateway`